From 59097210f25bfcabc82c9989d3c083a6febeadd6 Mon Sep 17 00:00:00 2001
From: Auger Eric <eric.auger@redhat.com>
Date: Tue, 4 Jun 2019 15:23:26 +0100
Subject: [PATCH 7/9] aarch64: Compile out IOH3420
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Auger Eric <eric.auger@redhat.com>
Message-id: <20190604152326.18510-1-eric.auger@redhat.com>
Patchwork-id: 88525
O-Subject: [RHEL-AV-8.1.0 qemu-kvm PATCH v2] aarch64: Compile out IOH3420
Bugzilla: 1627283
RH-Acked-by: Andrew Jones <drjones@redhat.com>
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

BZ: 1627283
BRANCH: rhel-av-8.1.0/master-4.0.0
UPSTREAM: N/A
BREW: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=21989313
TEST: On Sabre, no ioh3420 available

IOH3420 is not used on aarch64 or arm as we prefer the generic root port
so let's compile it out.

Signed-off-by: Eric Auger <eric.auger@redhat.com>

---

v1 -> v2
- do not touch arm-softmmu.mak

Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 default-configs/aarch64-rh-devices.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/default-configs/aarch64-rh-devices.mak b/default-configs/aarch64-rh-devices.mak
index 13ce7c7..224e29b 100644
--- a/default-configs/aarch64-rh-devices.mak
+++ b/default-configs/aarch64-rh-devices.mak
@@ -12,7 +12,6 @@ CONFIG_EDID=y
 CONFIG_FW_CFG_DMA=y
 CONFIG_GPIO_KEY=y
 CONFIG_I2C=y
-CONFIG_IOH3420=y
 CONFIG_IVSHMEM=y
 CONFIG_KVM=y
 CONFIG_LINUX=y
-- 
1.8.3.1

